src/events.ts

```diff
@@ -141,25 +141,33 @@ export interface EventSource {
   metadata?: Record<string, any>;
 }
 
+/**
+ * Represents seeking position.
+ */
```

Comment:
I don't think this offers any additional value. If you want to keep it anyways, I would prefer the following:
```suggestion
/**
 * Represents a seeking position.
 */
```